js/templates/modals/editListing/coupon.html

```diff
@@ -0,0 +1,39 @@
+<div class="col4 simpleFlexCol">
+  <% if (ob.errors['title']) print(ob.formErrorTmpl({ errors: ob.errors['title'] })) %>
+  <input type="text" class="clrBr clrP clrSh2 marginTopAuto" name="title" value="<%= ob.title %>" placeholder="<%= ob.polyT('editListing.coupons.titlePlaceholder') %>" maxlength=<%= ob.max.titleLength %>>
+</div>
+<div class="col4 simpleFlexCol">
+  <% if (ob.errors['discountCode']) print(ob.formErrorTmpl({ errors: ob.errors['discountCode'] })) %>
+  <input type="text" class="clrBr clrP clrSh2 marginTopAuto" name="discountCode" value="<%= ob.discountCode %>" placeholder="<%= ob.polyT('editListing.coupons.discountCodePlaceholder') %>">
+</div>
+<div class="col4 simpleFlexCol">
+  <% if (ob.errors['percentDiscount']) print(ob.formErrorTmpl({ errors: ob.errors['percentDiscount'] })) %>
+  <% if (ob.errors['priceDiscount']) print(ob.formErrorTmpl({ errors: ob.errors['priceDiscount'] })) %>
+  <div class="flexRow marginTopAuto">
+    <div class="inputSelect marginTopAuto">
+      <%
+        let discountAmount;
+
+        if (typeof ob.priceDiscount !== 'undefined') {
+          discountAmount = typeof ob.priceDiscount === 'number' ?
+            ob.formatPrice(ob.priceDiscount, ob.getCurrency() === 'BTC') :
+            ob.priceDiscount
+        } else {
+          discountAmount = typeof ob.percentDiscount === 'number' ?
+            parseFloat((ob.percentDiscount).toFixed(2)).toString() : ob.percentDiscount;
```

Comment:
I know this isn't really my domain here but from past experience it definitely think this  type of logic shout be encapsulated and reusable.